src/mediapipe_calculators/python_executor_calculator.cc

```diff
@@ -88,51 +114,67 @@ class PythonExecutorCalculator : public CalculatorBase {
             RET_CHECK(false);
         }
         nodeResources = it->second;
+        outputsTimestamp = mediapipe::Timestamp(mediapipe::Timestamp::Unset());
         LOG(INFO) << "PythonExecutorCalculator [Node: " << cc->NodeName() << "] Open end";
         return absl::OkStatus();
     }
 
     absl::Status Process(CalculatorContext* cc) final {
         LOG(INFO) << "PythonExecutorCalculator [Node: " << cc->NodeName() << "] Process start";
+        LOG(INFO) << "Input timestamp : " << cc->InputTimestamp();
+        for (const auto& input : cc->Inputs()) {
+            LOG(INFO) << input.Name() << " : " << !input.IsEmpty();
+        }
         py::gil_scoped_acquire acquire;
         try {
-            py::print("PYTHON: Acquired GIL");
+            LOG(INFO) << "PYTHON: Acquired GIL";
             if (pyIteratorPtr) {  // Iterator initialized
-                py::print("PyIterator initialized block");
+                if (receivedNewData(cc)) {
+                    LOG(INFO) << "[Node: " << cc->NodeName() << "] Node is already processing data. Create new stream for another request.";
+                    return absl::Status(absl::StatusCode::kResourceExhausted, "Node is already processing data. Create new stream for another request.");
```

Comment:
Isn't it better to use Ok status and ignore the next inputs to avoid the OVMS closure hang we still work on?

Reply:
Let's sync about this offline. All the catches at the end of the Process() also return non OK statuses and behave the same. So it seems we need to settle one approach for entire error handling.